doc/sneak.txt

```diff
@@ -364,12 +368,23 @@ Sneak uses these highlight groups:
 
 To customize colors: >
     highlight Sneak guifg=black guibg=red ctermfg=black ctermbg=red
+    highlight SneakCurrent guifg=black guibg=cyan ctermfg=black ctermbg=cyan
     highlight SneakScope guifg=red guibg=yellow ctermfg=red ctermbg=yellow
 
+    " Highlight current match the same way as other matches
+    highlight link SneakCurrent Sneak
+
+    " Highlight matches the same way as the / command
+    highlight link Sneak Search
+    highlight link SneakCurrent CurSearch
+
 To disable highlighting: >
     highlight link Sneak None
+    highlight link SneakCurrent None
+
     " Needed if a plugin sets the colorscheme dynamically:
-    autocmd User SneakLeave highlight clear Sneak
+    autocmd User SneakLeave highlight clear Sneak |
+                          \ highlight clear SneakCurrent
```

Comment:
With the `link` examples here, I noticed that if I wanna quickly try them through command mode i.e. `:hi link Sneak Search`, it requires a `!` after `hi`.

I was considering writing a note about that but also the error that you get could make you realize that if you know your vimscript:

```
E414: Group has settings, highlight link ignored

```